src/utils.ts

```diff
@@ -14,7 +14,7 @@ export enum MediaType {
 
 export function stripExtension(filename: string): string {
 	for (const ext of VIDEO_EXTENSIONS) {
-		const re = new RegExp(`\\.${ext}$`);
+		const re = new RegExp(`${ext}$`);
```

Comment:
Yep, you are absolutely correct, good spot.

Reply:
I've never seen/used that package before, imo I think that adding additional packages to fix a single line is outside the scope of this PR.

It would be better and probably more performant to replace regex with endsWith but that's something I'd rather let the main developers decide as maybe there is a reason for using regex in this function.